bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -1354,18 +1371,28 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 </finidat>
 
 <!-- CTSM6_0 - use clm6_0_GSWP3v1 BgcCrop at f09 for all clm5_1/clm6_0 options -->
-<!-- use_init_interp true because the file was created with ctsm5.3 datasets -->
+<!-- TODO slevis VALID COMMENT? use_init_interp true because the file was created with ctsm5.3 datasets -->
 <finidat hgrid="0.9x1.25" mask="gx1v7" use_cn=".true." use_cndv=".false." use_fates=".false."
          ic_ymd="18500101" sim_year="1850" do_transient_pfts=".false." use_excess_ice=".true."
          ic_tod="0" glc_nec="10" use_crop=".true." irrigate=".false."
          phys="clm5_1" use_init_interp=".true."
```

Comment:
We should be able to remove use_init_interp=.true. part here...

Reply:
This is the only one that accepted the removal and then also worked. See relevant discussion elsewhere, so I'm resolving this one.